test/integration/buildscript_int_test.go

```diff
@@ -168,9 +168,8 @@ func (suite *BuildScriptIntegrationTestSuite) TestBuildScriptRequirementVersionA
 	cp := ts.Spawn("config", "set", constants.AsyncRuntimeConfig, "true")
 	cp.ExpectExitCode(0)
 
-	// Enable as part of DX-3230.
-	//cp = ts.Spawn("config", "set", constants.OptinBuildscriptsConfig, "true")
-	//cp.ExpectExitCode(0)
+	cp = ts.Spawn("config", "set", constants.OptinBuildscriptsConfig, "true")
+	cp.ExpectExitCode(0)
```

Comment:
This intentional for this PR?

Reply:
Yes, there was a removed comment indicated this ought to be enabled now. Previously there was a parse error, but this PR fixes that, so enabling this verifies the fix.